common/maven/src/main/java/io/helidon/build/common/maven/MavenCommand.java

```diff
@@ -499,6 +502,18 @@ public Builder afterShutdown(Runnable afterShutdown) {
             return this;
         }
 
+        /**
+         * Sets the maven executable path to be used for command execution.
+         *
+         * @param mvn the maven executable path
+         * @return This builder.
+         */
+        public Builder executable(Path mvn) {
+            customMavenExec = mvn.toString();
```

Comment:
Rename `customMavenExec` to `executable` to match the builder method.
Also you be able to replace `ignoreMavenVersion` with `executable != null`